packages/compass-editor/src/editor.tsx

```diff
@@ -877,8 +877,10 @@ const BaseEditor = React.forwardRef<EditorRef, EditorProps>(function BaseEditor(
             key: 'Ctrl-Shift-c',
             run: copyAll,
           },
-          ...defaultKeymap,
+          // Close brackets keymap overrides default backspace handler for
+          // matching bracket case
```

Comment:
Was double-checking something in codemirror docs for my quick poc for this and noticed that it was already a feature, we just misconfigured it a bit